.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
```

Comment:
I don't understand this extra step because ubuntu-latest already has python3?

Reply:
Maybe it's to make `python` (rather than `python3`) work, regardless of what version is installed?
I'll try without it though.

Reply:
Removed. Had to switch command to use `python3`